pyartcd/pyartcd/locks.py

```diff
@@ -15,6 +15,7 @@ class Lock(enum.Enum):
     BUILD = 'lock:build:{version}'
     MASS_REBUILD = 'lock:mass-rebuild-serializer'
     SIGNING = 'lock:signing:{signing_env}'
+    BUILD_SYNC = 'lock:build-sync:{version}'
```

Comment:
So I think we need version-assembly here, running build-sync on 4.15 stream and say, 4.15.0-rc.1 parallely should be fine

Reply:
ah I see we only use this when assembly is stream